build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile "com.fasterxml.jackson.core:jackson-core:2.9.9"
     compile "com.fasterxml.jackson.core:jackson-databind:2.9.9"
     compile group: 'net.sf.jtidy', name: 'jtidy', version: 'r938'
+    compile files('./lib/CombineArchive-1.4.0-jar-with-dependencies.jar')
```

Comment:
A maven repository exists for the COMBINEArchive (also it is a bit hidden). Use the maven version instead of adding the library directly to the project
- https://mvn.bio.informatik.uni-rostock.de/releases/de/unirostock/sems/CombineArchive/
- https://sems.bio.informatik.uni-rostock.de/2013/10/maven-repository/

Reply:
I am not able to find the repo. Please see https://mvn.bio.informatik.uni-rostock.de/releases/de/binfalse/
Combine-Archive is not available here.

Reply:
There is also https://bin.bio.informatik.uni-rostock.de/CombineArchive/

Reply:
It is documented in the the links I added above:
1. add the custom maven repository
```
<repositories>
  <repository>
    <id>sems-maven-repository-releases</id>
    <name>SEMS Maven Repo</name>
    <url>http://mvn.sems.uni-rostock.de/releases/</url>
    <layout>default</layout>
    <snapshots>
      <enabled>false</enabled>
    </snapshots>
  </repository>
  <repository>
    <id>sems-maven-repository-snapshots</id>
    <name>SEMS Maven Repo</name>
    <url>http://mvn.sems.uni-rostock.de/snapshots/</url>
    <layout>default</layout>
    <releases>
      <enabled>false</enabled>
    </releases>
  </repository>
</repositories>
```
2. add the maven dependency
```
        <dependency>
            <groupId>de.unirostock.sems</groupId>
            <artifactId>CombineArchive</artifactId>
            <version>1.3.0</version>
        </dependency>
```

Reply:
This is solved, dependency is correct now